### ISSUE TITLE
[Accessibility] Color Contrast is not respected. 

### ISSUE THREAD

**Issue by andreacantone** [ankidroid/Anki-Android#7717]
###### Reproduction Steps

1. Activate the app [Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=it&gl=US)
2. Open the app Anki-Android.
3. Activate the registration from the Accessibility Scanner and use the Anki-Android app.
4. After using the app, stop the recording.

###### Expected Result

The contrast ratio between text and background should be at least 4.5 .

###### Actual Result

Some elements don't respect the contrast ratio.

![IMG_20201118_175454](https://user-images.githubusercontent.com/25429624/99562659-aac27200-29c8-11eb-9643-ce018670ba89.png)

###### Research
*Enter an [x] character to confirm the points below:*

- [x] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid
- [x] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [x] I have searched for similar existing issues here and on the user forum
- [ ] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))
